recipes-domu/domu-image-fusion/files/50_local.conf.inc

```diff
@@ -0,0 +1,4 @@
+# XXX: disable smack security so we can boot with nfsroot
+# FIXME: at the moment this is the first occurance of BBMASK
+# for the build, so we cannot use .= syntax here as configuration fails
+BBMASK = "meta-intel-iot-security/meta-security-smack/recipes-kernel/linux"
```

Comment:
Whoa! If this is a problem, then you can add empty `BBMASK` somewhere else and then extend it there.
Because, otherwise, this change will cause problems in the future.

Reply:
1. AGL build doesn't set this to empty value and I am not sure if this is only the case for the configuration/target I use or it is common for all. Thus I take an action and set it
2. The only way to set it to empty string is either patch AGL or provide it in this file